project/src/components/tabs/details.jsx

```diff
@@ -12,7 +12,7 @@ function DetailsTab({currentFilm}) {
         </p>
         <p className="film-card__details-item">
           <strong className="film-card__details-name">Starring</strong>
-          <span className="film-card__details-value">{currentFilm.starring.map((item) => <span key={item}>{item}<br/></span>)}
+          <span className="film-card__details-value">{Array.isArray(currentFilm.starring) ? currentFilm.starring.map((item) => <span key={item}>{item}<br/></span>) : ''}
```

Comment:
Более короткий способ записи такой:

```suggestion
          <span className="film-card__details-value">{Array.isArray(currentFilm.starring) && currentFilm.starring.map((item) => <span key={item}>{item}<br/></span>)}
```